pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -347,6 +353,19 @@ class PillarboxCastPlayer internal constructor(
         }
     }
 
+    override fun handleSetPlaybackParameters(playbackParameters: PlaybackParameters) = withRemoteClient {
+        val playbackRate = playbackParameters.speed.toDouble().coerceIn(MediaLoadOptions.PLAYBACK_RATE_MIN, MediaLoadOptions.PLAYBACK_RATE_MAX)
+
+        setPlaybackRate(playbackRate)
+    }
+
+    override fun handleSetPlaylistMetadata(playlistMetadata: MediaMetadata): ListenableFuture<*> {
+        this@PillarboxCastPlayer.playlistMetadata = playlistMetadata
+        invalidateState()
```

Comment:
Maybe check if the new value is different before calling invalidateState?

Reply:
Done